src/Github/CreateChangelogText.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\AutomaticReleases\Github;
+
+use Doctrine\AutomaticReleases\Github\Api\GraphQL\Query\GetMilestoneChangelog\Response\IssueOrPullRequest;
+use Doctrine\AutomaticReleases\Github\Api\GraphQL\Query\GetMilestoneChangelog\Response\Label;
+use Doctrine\AutomaticReleases\Github\Api\GraphQL\Query\GetMilestoneChangelog\Response\Milestone;
+use Psr\Http\Message\UriInterface;
+
+final class CreateChangelogText
```

Comment:
I'd like to see https://github.com/jwage/changelog-generator/blob/master/src/ChangelogGenerator/ChangelogGenerator.php reused here, it makes the output more consumer-friendly.

Reply:
Possibly to be considered for later/separate patch. The `Milestone` abstraction I built so far is quite simplistic, but sufficient for now.